src/index.ts

```diff
@@ -46,7 +42,9 @@ async function run() {
 
   log.info(`Loaded ${personaCount} personas for the bot.`)
 
-  let potd = loadPotd({ forcePersonaId: args.forcePersona })
+  client.commands = new Collection()
+  const loadedCommands = await loadCommands()
+  loadedCommands.map(c => client.commands?.set(c.data.name, c))
```

Comment:
Move commands loading after personas import, so that the change-persona command is able to retrieve the personas